### DISCUSSION TITLE
ECAM SD switches

### DISCUSSION THREAD

**Discussion by cebeuh** [flybywiresim/aircraft#6352] [Help]
Hello which API function I can use to switch the ECAM SD pages since A32NX_ECAM_SD_CURRENT_PAGE_INDEX only change the button indication and not the page itself? thank you and best regards

**Comment by Muyz** [comment]
This also interests me; implementing SW to use with HW in my home cockpit. Does anyone know?

**Reply by Muyz** [reply]
Answer:

Use HVars:

H:A320_Neo_EICAS_2_ECAM_CHANGE_PAGE_ENG
H:A320_Neo_EICAS_2_ECAM_CHANGE_PAGE_BLEED
H:A320_Neo_EICAS_2_ECAM_CHANGE_PAGE_PRESS
etc.
